poetry.lock

```diff
@@ -127,7 +127,7 @@ optional = false
 python-versions = ">=3.6.0"
 
 [package.extras]
-unicode-backport = ["unicodedata2"]
+unicode_backport = ["unicodedata2"]
```

Comment:
for my knowledge I see you replaced multiple package naming from - to _ is it a new issue, a coding convention or something ?

Reply:
anything in the .lock file is completely automated and managed by `poetry`. So my guess is that they are doing something with the naming.